persistent-volumes/nfs-internal-ob/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
@kdanW you don't need to commit any of the resources related to the NFS provisioner including the packaged Helm chart. Managing this during a deployment can be handled via the `requirements.yaml`.

Reply:
@chirangaalwis for that we will have to add the MySQL-OB chart and NFS-OB chart to WSO2 Helm repository right?

Reply:
@kdanW no you need not. Use the original, stable NFS Server Provisioner [1] for this purpose.

Please see the following sample `requirements.yaml` I've created for WSO2 API Management pattern 2.

```
dependencies:
  - name: mysql-am
    version: "2.6.0-7"
    repository: "https://helm.wso2.com"
    condition: wso2.deployment.dependencies.mysql
  - name: nfs-server-provisioner
    version: "0.3.0"
    repository: "https://kubernetes-charts.storage.googleapis.com"
    condition: wso2.deployment.dependencies.nfsProvisioner.enabled
```

[1]: [NFS Server Provisioner](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner)

Reply:
@chirangaalwis i need to set some custom values to ```nfs-server-provisioner chart``` as in https://github.com/wso2/kubernetes-open-banking/pull/22/files#diff-55ac794426d471774fccfd3adbcce21f . Can we do this if we add the stable chart in ```requirments.yaml``` directly?

Reply:
@kdanW if you intend to change the customize the `values.yaml` of NFS Server Provisioner, you need to repackage the Chart as in MySQL OB Helm Chart. AFAIK, we cannot override values in the `requirements.yaml`. Please investigate on this, further.

What custom value changes, do we need to make?

Reply:
Yes @chirangaalwis , That is the process that I have followed so far. I need to set the values in the set the custom values in the Values.yaml in [1].

[1] https://github.com/wso2/kubernetes-open-banking/pull/22/files#diff-55ac794426d471774fccfd3adbcce21f

Reply:
Fixed in https://github.com/wso2/kubernetes-open-banking/pull/22/commits/801acbc3a7226e2aa43f6fbdeffb5f723f740695